tests/interp_mod_grib2.F90

```diff
@@ -64,9 +64,11 @@ subroutine interp(grid, interp_opt)
 
```

Comment:
Can you please name all NCEPLIBS tests "test_SOMETHING.F90" for Fortran and "tst_SOMETHING.c" for C.

So all Fortran test programs should have a name "test_*.F90".